exercises/etl/canonical-data.json

```diff
@@ -0,0 +1,62 @@
+{
+  "transform": {
+    "description": "transforms the input map from a map of value: [ keys ] to a map of key: value for each of the keys",
+    "cases": [
+      {
+        "description": "transforms one value",
+        "input": {
+          "1": ["A"]
+        },
+        "expected": {
+          "a" : 1
+        }
+      },
+      {
+        "description": "tramsforms more values",
```

Comment:
Do you have an idea for the test descriptions?  I started with something more like
`transforms a map with a single key and an array of one value into a map with one key and one value`.  That works for the first one, but the next ones get pretty wordy.  Is that where you thought that might go?

If this file acted like I think of rspec describe/context/it blocks, the outer description could be kind of the prefix to the inner descriptions with something like

```
{
  "transform": {
    "description": "transforms the input map",
    "cases": [
      {
        "description": "with one key that has an array with one value into a map with  one key and one value",
        "input": {"1": ["A"] },
        "expected": { "a" : 1 }
      },
      {
        "description": "with one key and multiple values into a map with multiple keys each with a single value",
        "input": { "1": ["A", "E", "I", "O", "U" ] },
        "expected": {
          "a" : 1,
          ...
```

I'm happy to rewrite the descriptions, but i'd love a little direction or suggestion